Re-import Trusted Types WPTs

lukewarlow opened the pull request:
#### 7b289723078848a337ba56b2f1f5262f477200c8
<pre>
Re-import Trusted Types WPTs
<a href="https://bugs.webkit.org/show_bug.cgi?id=289903">https://bugs.webkit.org/show_bug.cgi?id=289903</a>

Reviewed by Anne van Kesteren.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/a50e17d65c4855406afa14957e6388b1db3247f0">https://github.com/web-platform-tests/wpt/commit/a50e17d65c4855406afa14957e6388b1db3247f0</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DOMWindowTimers-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-block-eval-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-block-eval-function-constructor.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-DedicatedWorker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-DedicatedWorker.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-worker-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-SharedWorker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-worker-constructor-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-SharedWorker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-worker-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-eval-expected.txt: Renamed from LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-eval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-importScripts-expected.txt: Renamed from LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-importScripts.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-setTimeout-setInterval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Document-write-exception-order.xhtml:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Element-setAttribute-respects-Elements-node-documents-globals-CSP-after-adoption-from-non-TT-realm-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Element-setAttribute-respects-Elements-node-documents-globals-CSP-after-adoption-from-non-TT-realm.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-block-eval-function-constructor.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-block-eval-function-constructor.https.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-eval.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-eval.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-importScripts.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-importScripts.https.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-DedicatedWorker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-DedicatedWorker.https.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-ServiceWorker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-ServiceWorker.https.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-SharedWorker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-SharedWorker.https.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-block-eval-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-block-eval-function-constructor.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-constructor.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-constructor.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-eval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-eval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-importScripts-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-importScripts.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-setTimeout-setInterval.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicy-createXXX.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-none-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-getAttributeType-event-handler-content-attributes.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-getAttributeType-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-getAttributeType.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-block-eval-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-block-eval-function-constructor.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DedicatedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DedicatedWorker-setTimeout-setInterval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Document-write.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Element-setAttribute.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-HTMLElement-generic.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-SharedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-SharedWorker-setTimeout-setInterval.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Window-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Window-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-script-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-script-element.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-svg-script-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-svg-script-element.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-TypeError-belongs-to-the-global-object-realm-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-TypeError-belongs-to-the-global-object-realm.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-report-only-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/resources/block-text-node-insertion.js:
(string_appeared_here.checkMessage):
(string_appeared_here.checkMessage.): Deleted.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-mutations-in-callback.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-mutations-in-callback.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-no-require-trusted-types-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-no-require-trusted-types.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-default-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-default-policy.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-no-default-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-no-default-policy.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/WorkerGlobalScope-importScripts.https.js:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/attributes.js: Added.
(createTrustedOutput):
(findAttribute):
(const.attributeSetterData.runSetter):
(return.element.setAttribute.runSetter):
(return.element.setAttribute):
(return.element.setAttributeNS.runSetter):
(return.element.setAttributeNS):
(return.element.setAttributeNode.runSetter):
(return.element.setAttributeNode):
(return.element.setAttributeNodeNS.runSetter):
(return.element.setAttributeNodeNS):
(return.nodeMap.setNamedItem.runSetter):
(return.nodeMap.setNamedItem):
(return.nodeMap.setNamedItemNS.runSetter):
(return.nodeMap.setNamedItemNS):
(return.runSetter):
(return):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/block-eval-function-constructor.js:
(createScript):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/create-trusted-type-policies.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/csp-violations.js:
(trusted_type_violations_and_exception_for):
(tryCreatingTrustedTypePoliciesWithCSP):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/send-plain-string-to-trusted-type-sink.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DOMWindowTimers-setTimeout-setInterval.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-setTimeout-setInterval-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-setTimeout-setInterval-worker.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DOMParser-parseFromString-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-execCommand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-parseHTMLUnsafe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-insertAdjacentHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setAttribute-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setHTMLUnsafe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLIFrameElement-srcdoc-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Range-createContextualFragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-setHTMLUnsafe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-source-file-path-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-svg-script-set-href-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/worker-constructor.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/worker-constructor.https.html: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-none-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-report-only-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002-expected.txt: Added.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DOMParser-parseFromString-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-execCommand-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-parseHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-write-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-insertAdjacentHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setAttribute-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLIFrameElement-srcdoc-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Range-createContextualFragment-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-source-file-path-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-svg-script-set-href-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-none-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes-expected.txt: Added.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-report-only-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative-expected.txt:
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DOMParser-parseFromString-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-execCommand-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-parseHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-write-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-insertAdjacentHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setAttribute-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLIFrameElement-srcdoc-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Range-createContextualFragment-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-source-file-path-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-svg-script-set-href-expected.txt: Removed.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/292363@main">https://commits.webkit.org/292363@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/53ed2aa607e40f67d1f261fe9bf5d5db5601aeeb

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/95595 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/15197 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/5106 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/100644 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/46100 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/15483 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/23634 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/72909 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/30169 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/98598 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/11607 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/86345 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/53241 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/11313 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/4058 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/45436 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/81502 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/4177 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/102679 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/22645 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/16541 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/81948 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/22897 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/82359 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/81299 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20419 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/25883 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/3352 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/15990 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/22613 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/22272 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/25748 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/24014 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
That's a surprising amount of changes.

lukewarlow committed:
Re-import Trusted Types WPTs
https://bugs.webkit.org/show_bug.cgi?id=289903

Reviewed by Anne van Kesteren.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/a50e17d65c4855406afa14957e6388b1db3247f0

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DOMWindowTimers-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-block-eval-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-block-eval-function-constructor.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-DedicatedWorker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-DedicatedWorker.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-worker-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-SharedWorker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-worker-constructor-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor-from-SharedWorker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-worker-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-constructor.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-eval-expected.txt: Renamed from LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-eval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-importScripts-expected.txt: Renamed from LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-importScripts.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/DedicatedWorker-setTimeout-setInterval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Document-write-exception-order.xhtml:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Element-setAttribute-respects-Elements-node-documents-globals-CSP-after-adoption-from-non-TT-realm-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Element-setAttribute-respects-Elements-node-documents-globals-CSP-after-adoption-from-non-TT-realm.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-block-eval-function-constructor.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-block-eval-function-constructor.https.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-eval.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-eval.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-importScripts.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorker-importScripts.https.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-DedicatedWorker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-DedicatedWorker.https.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-ServiceWorker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-ServiceWorker.https.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-SharedWorker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-SharedWorker.https.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-Worker.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-block-eval-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-block-eval-function-constructor.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-constructor.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-constructor.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-eval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-eval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-importScripts-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-importScripts.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/SharedWorker-setTimeout-setInterval.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicy-createXXX.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-none-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-getAttributeType-event-handler-content-attributes.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-getAttributeType-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-getAttributeType.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-block-eval-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-block-eval-function-constructor.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Window-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-eval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-eval-function-constructor-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DedicatedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DedicatedWorker-setTimeout-setInterval.html: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Document-write.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Element-setAttribute.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-HTMLElement-generic.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-SharedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-SharedWorker-setTimeout-setInterval.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Window-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Window-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-script-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-script-element.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-svg-script-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-text-node-insertion-into-svg-script-element.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-TypeError-belongs-to-the-global-object-realm-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-TypeError-belongs-to-the-global-object-realm.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-report-only-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/resources/block-text-node-insertion.js:
(string_appeared_here.checkMessage):
(string_appeared_here.checkMessage.): Deleted.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-mutations-in-callback.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-mutations-in-callback.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-no-require-trusted-types-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-no-require-trusted-types.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-default-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-default-policy.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-no-default-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-attributes-require-trusted-types-no-default-policy.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/WorkerGlobalScope-importScripts.https.js:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/attributes.js: Added.
(createTrustedOutput):
(findAttribute):
(const.attributeSetterData.runSetter):
(return.element.setAttribute.runSetter):
(return.element.setAttribute):
(return.element.setAttributeNS.runSetter):
(return.element.setAttributeNS):
(return.element.setAttributeNode.runSetter):
(return.element.setAttributeNode):
(return.element.setAttributeNodeNS.runSetter):
(return.element.setAttributeNodeNS):
(return.nodeMap.setNamedItem.runSetter):
(return.nodeMap.setNamedItem):
(return.nodeMap.setNamedItemNS.runSetter):
(return.nodeMap.setNamedItemNS):
(return.runSetter):
(return):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/block-eval-function-constructor.js:
(createScript):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/create-trusted-type-policies.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/csp-violations.js:
(trusted_type_violations_and_exception_for):
(tryCreatingTrustedTypePoliciesWithCSP):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/send-plain-string-to-trusted-type-sink.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DOMWindowTimers-setTimeout-setInterval.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-setTimeout-setInterval-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-setTimeout-setInterval-worker.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DOMParser-parseFromString-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-execCommand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-parseHTMLUnsafe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-insertAdjacentHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setAttribute-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setHTMLUnsafe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLIFrameElement-srcdoc-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Range-createContextualFragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-setHTMLUnsafe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-setTimeout-setInterval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-setTimeout-setInterval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-source-file-path-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-svg-script-set-href-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/worker-constructor.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/worker-constructor.https.html: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-none-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-report-only-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002-expected.txt: Added.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DOMParser-parseFromString-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-execCommand-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-parseHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-write-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-insertAdjacentHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setAttribute-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLIFrameElement-srcdoc-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Range-createContextualFragment-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-innerHTML-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-source-file-path-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/trusted-types-svg-script-set-href-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-cspTests-none-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/inheriting-csp-for-local-schemes-expected.txt: Added.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/require-trusted-types-for-report-only-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative-expected.txt:
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DOMParser-parseFromString-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-execCommand-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-parseHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Document-write-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-insertAdjacentHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setAttribute-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLIFrameElement-srcdoc-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Range-createContextualFragment-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-innerHTML-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ShadowRoot-setHTMLUnsafe-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-source-file-path-expected.txt: Removed.
* LayoutTests/platform/win/imported/w3c/web-platform-tests/trusted-types/trusted-types-svg-script-set-href-expected.txt: Removed.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/292363@main
